Source/minitext.cpp

```diff
@@ -97,56 +97,56 @@ void __cdecl DrawQTextBack()
 void __fastcall PrintQTextChr(int screen_x, int screen_y, char *cel_buf, int frame)
 {
 	char *v4;         // ebx
-	char *v5;         // esi
-	char *v6;         // edi
+	char *celGlyph;         // esi

```

Comment:
Normally it's better not to change names like this. We try to stick to the original names as found in assert strings from the debug and PSX sym files.
You can find the PSX converted to C here:
https://github.com/diasurgical/scalpel/tree/master/psx/_dump_/_dump_merge_c_src_/diabpsx/source
As it happens they don't contain the PrintQTextChr function, but there might be similar functions in minitext.cpp that will give an idea of what naming convention was used. Some of the variables might also go away once the function is cleaned up, so I would recommend deferring changes to for a PR that specifically deals with cleaning up this function.

Reply:
Some of the assert strings can be found here: https://github.com/nomdenom/devil-beta/blob/master/Source/minitext.cpp
Again there isn't much to find for this function.